app/controllers/organizations/new.js

```diff
@@ -240,10 +269,13 @@ export default class OrganizationsNewController extends Controller {
       identifierKBO.validate(),
       identifierSharepoint.validate(),
     ]);
+    this.attemptedSave = true;
 
     if (
       this.features.isEnabled('edit-contact-data') ||
-      this.currentOrganizationModel.isPrivateOcmwAssociation
+      this.currentOrganizationModel.isPrivateOcmwAssociation ||
+      this.currentOrganizationModel.isAssociationOther ||
+      this.currentOrganizationModel.isCorporationOther
```

Comment:
**Suggestion:** Instead of redefining the condition every time, can we create a utils function `isContactEditableOrganization`.